<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab 5 - 2D OpenGL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="triangle">
     <attribute name="title">
      <string>VAOs + VBOs</string>
     </attribute>
     <widget class="QGroupBox" name="shapeGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>142</width>
        <height>332</height>
       </rect>
      </property>
      <property name="title">
       <string>Shape</string>
      </property>
      <widget class="QCheckBox" name="linesEnabled">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Draw lines</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="triangleCanvas" native="true">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>29</y>
        <width>321</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fragShaders">
     <attribute name="title">
      <string>Fragment Shaders</string>
     </attribute>
     <widget class="QWidget" name="fragCanvas" native="true">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>275</width>
        <height>275</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="shaderProgram">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Shader Program</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="solidShader">
          <property name="text">
           <string>Solid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="gradientShader">
          <property name="text">
           <string>Gradient</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="textureShader">
          <property name="text">
           <string>Texture</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
